code/go/0chain.net/chaincore/smartcontractinterface/interface.go

```diff
@@ -32,11 +32,11 @@ type SmartContractTransactionData struct {
 
 type SmartContractInterface interface {
 	Execute(t *transaction.Transaction, funcName string, input []byte, balances c_state.StateContextI) (string, error)
+	SetSC(sc *SmartContract, bc BCContextI)
```

Comment:
This is reverting a previous PR https://github.com/0chain/0chain/pull/313 from 21 days ago. There are others examples.